content/_index.md

```diff
@@ -15,6 +15,7 @@ The following Rust projects already integrate AccessKit:
 * [egui](https://github.com/emilk/egui)
 * [Slint](https://slint.dev/)
 * [Xilem](https://github.com/linebender/xilem)
+* [Freya](https://freyaui.dev/)
```

Comment:
I'd prefer to keep this list alphabetically sorted. We probably don't want to give the impression that AccessKit website authors favor some UI toolkits over others.

Reply:
ohh sure!